### DISCUSSION TITLE
blog/hello-google-bard/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#325] [Announcements]
# blog/hello-google-bard/

这篇文章介绍了 Google Bard，它是一个由 Google 开发的生成式 AI 工具，可以与用户合作，帮助他们更有效地思考和创造。文章还比较了 Bing Chat、OpenAI ChatGPT 和 Google Bard 的不同，以及它们对同一个问题的回答。笔者认为必应聊天的回答最好。

https://jimmysong.io/blog/hello-google-bard/

**Comment by moreoronce** [comment]
目前来看new bing的体验应该是最好的，bard的体验是最糟糕的，如果不算文心一言的话

**Reply by rootsongjc** [reply]
确实，还不如 perplexity

**Comment by rootsongjc** [comment]
这篇文章已经过时了，已经被 Gemini 替换掉了，不过我不是很习惯用 Gemini，只有在 GitHub Copilot 中使用 Gemini 2.5 pro agent 编程效果还不错。跟它对话时，感觉结果不如 ChatGPT。
